app.js

```diff
@@ -3,21 +3,15 @@ const serverless = require('serverless-http')
 const bodyParser = require('body-parser')
 const axios = require('axios')
 
-const maintenance = require('./modules/maintenance')
-const deploy = require('./modules/deploy')
-const report = require('./modules/report')
+
+const { verifySlack } = require('./modules/middleware')
 const diff = require('./modules/diff')
 const food = require('./modules/food')
-const boardroom = require('./modules/boardroom')
```

Comment:
why are these modules being removed?

Reply:
because they're not effective, or not used anymore